amoc_controller rework (#114)
esl/amoc: doc/distributed-run.md

and start scenarios on all known nodes (except master).

```erlang
amoc_dist:do(my_scenario, 100, Settings).
```

Start `my_scenario` spawning 100 amoc users with IDs from the range [1,100] inclusive.
In this case sessions are going to be distributed across all nodes except master.

> `Settings` is an optional proplist with scenario options that can be extracted using amoc_config module,

Reviewer: ```suggestion
`Settings` is an optional proplist with scenario options that can be extracted using amoc_config module.
```

>  values provided in this list shadows OS and APP environment variables. also this settings will be propagated

Reviewer: ```suggestion
The values provided in this list shadow OS and APP environment variables. Note that these settings will be propagated
```

>  automatically withing all the nodes in the amoc cluster.

Reviewer: ```suggestion
 automatically among all the nodes in the amoc cluster.
```

Add 50 more users to the currently started scenario.

```erlang
amoc_dist:remove(50, Force).
```

Remove 50 sessions. 

> Where ``Force`` is the boolean value, that has the next meaning:

Reviewer: ```suggestion
Where ``Force`` is a boolean of value:  
```

> * ``true``  - kill the user processes using ``supervisor:terminate_child/2`` function

Reviewer: ```suggestion
* ``true``  - to kill the user processes using ``supervisor:terminate_child/2`` function
```

> * ``false`` - just send ``exit(User,shutdown)`` signal to the user process (can be ignored by the user)

Reviewer: ```suggestion
* ``false`` - to send ``exit(User,shutdown)`` signal to the user process (can be ignored by the user)
```

All the users are ``temporary`` children of the ``simple_one_for_one`` supervisor with the ``shutdown`` 
key set to ``2000``.

> Also all the user processes trap exit signal.

Reviewer: ```suggestion
Also all the users process a trap exit signal.
```
??

Author: nope, here trap is a verb
